onnx/shape_inference/implementation.cc

```diff
@@ -426,7 +427,11 @@ static void InferShapesImpl(
         materializeSymbolicShape(inferredType, symbolTable);
         // Now we can merge pre-existing and inferred info
         mergeShapesAndTypes(*inferredType, existingType);
-
+        if (schema->has_data_propagation_function()) {
```

Comment:
whether to enable data propagation or not needs to be controlled by an option... we can talk about it offline

Reply:
I added it as "data_prop" (any other idea?). Since the function now is not complete, setting it as an option (default is False) makes more sense to me. Please let me know if you have other concern. Thanks